frontend/public/components/nav/nav-header.tsx

```diff
@@ -55,25 +71,43 @@ const NavHeader_: React.FC<NavHeaderProps & StateProps> = ({ onPerspectiveSelect
     [isPerspectiveDropdownOpen, togglePerspectiveOpen],
   );
 
-  const perspectiveItems = React.useMemo(
-    () =>
-      perspectiveExtensions.map((nextPerspective: Perspective) => (
+  const perspectiveItems = React.useMemo(() => {
+    const items = perspectiveExtensions.map((nextPerspective: Perspective) => (
+      <DropdownItem
+        key={nextPerspective.properties.id}
+        onClick={(event: React.MouseEvent<HTMLLinkElement>) =>
+          onPerspectiveSelect(event, nextPerspective)
+        }
+        isHovered={nextPerspective.properties.id === activePerspective}
+        component="button"
+      >
+        <Title headingLevel="h2" size="md" data-test-id="perspective-switcher-menu-option">
+          <span className="oc-nav-header__icon">{nextPerspective.properties.icon}</span>
+          {nextPerspective.properties.name}
+        </Title>
+      </DropdownItem>
+    ));
+    if (acmLink) {
+      items.push(
         <DropdownItem
-          key={nextPerspective.properties.id}
-          onClick={(event: React.MouseEvent<HTMLLinkElement>) =>
-            onPerspectiveSelect(event, nextPerspective)
-          }
-          isHovered={nextPerspective.properties.id === activePerspective}
+          key={ACM_LINK_ID}
+          onClick={() => {
+            window.location.href = acmLink.spec.href;
+          }}
+          isHovered={ACM_LINK_ID === activePerspective}
           component="button"
```

Comment:
Should this be an `a` element instead of a `button`?

Reply:
From an earlier thread:  _"I made it an `a` and the cursor changed from a pointer to an I icon, other than that behaved the same."_

Reply:
Right, so let's make it an `a` :) We can set the pointer style if needed to make it consistent since you wouldn't normally select the text here.

Reply:
ok, changed to an `a` and added `cursor: pointer`